packages/core/src/types/arn/ses/index.ts

```diff
@@ -0,0 +1 @@
+export * from './SESArn';
```

Comment:
the file name is not capitalized well. Be careful, it can sometimes work in local but fail when building in the CI